be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/MethodValidator.scala

```diff
@@ -0,0 +1,23 @@
+package ch.epfl.pop.pubsub.graph.validators
+
+import ch.epfl.pop.model.network.JsonRpcRequest
+import ch.epfl.pop.pubsub.graph.{GraphMessage, PipelineError}
+
+object MethodValidator extends MethodContentValidator {
```

Comment:
I would expect `MethodValidator` to validate a method.
Am I correct in understanding here we're validating the parameters?
Could we rename it as ParameterValidator?

Reply:
LGTM